src/components/Table/TableRow/TableRow.module.css

```diff
@@ -0,0 +1,7 @@
+.row:nth-of-type(even) td {
```

Comment:
You are styling elements not in this component.
Is it possible to leave td away?:
```css
.row:nth-of-type(even) {}
```

Reply:
You can set the background right away on the row:
https://www.w3schools.com/css/tryit.asp?filename=trycss_table_color

Reply:
Yes, but then I won't be able to a set a border radius.